src/RestApiClient.js

```diff
@@ -128,6 +128,7 @@ class RestApiClient {
    *
    * @param  {Object} query
    * @return {String}
+   * Uses code from https://stackoverflow.com/a/42604801/4422345 for this function.
```

Comment:
Lil' nitpick, would you mind moving this above the docblock tags? Or perhaps putting it as a [`@see`](https://phpdoc.org/docs/latest/references/phpdoc/tags/see.html) tag?